proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -645,6 +645,7 @@ message ShardReplicationStatusPerCluster {
 message RebuildMutableStateRequest {
     string namespace_id = 1;
     temporal.api.common.v1.WorkflowExecution execution = 2;
+    bytes branch_token = 3;
```

Comment:
if there is plan to extend the usage of this, then just call it token?

Reply:
since import workflow execution API is landed, i will remove this field / functionality